File: Java/Elektronik.java
public class Elektronik {
    protected int id;
    protected String nama;
    protected String merek;
    protected String harga;
    protected String gambar;

    public Elektronik() {
        this.id = 0;
        this.nama = "";
        this.merek = "";
        this.harga = "";
        this.gambar = "";
    }

    public Elektronik(int id, String nama, String merek, String harga, String gambar) {
        this.id = id;
        this.nama = nama;
        this.merek = merek;
        this.harga = harga;
        this.gambar = gambar;
    }
}

File: CPP/Elektronik.cpp
#include <iostream>
#include <iomanip>
#include <string>
using namespace std;

class Elektronik {
private:
    int id;
    string nama, merek, harga, gambar;

public:
    Elektronik() : id(0), nama(""), merek(""), harga(""), gambar("") {}
    Elektronik(int i, string n, string m, string h, string g)
        : id(i), nama(n), merek(m), harga(h), gambar(g) {}

    int getId() const{ 
        return id; 
    }
    string getNama() const{ 
        return nama; 
    }
    string getMerek() const{ 
        return merek; 
    }
    string getHarga() const{ 
        return harga; 
    }
    string getGambar() const{ 
        return gambar; 
    }

    void setId(int i){ 
        id = i; 
    }
    void setNama(string n){ 
        nama = n; 
    }
    void setMerek(string m){ 
        merek = m; 
    }
    void setHarga(string h){ 
        harga = h; 
    }
    void setGambar(string g){ 
        gambar = g; 
    }
};

